.factory/automation.yml

```diff
@@ -110,13 +110,7 @@ build:
     test-example-python:
       image: vaticle-ubuntu-21.04
       type: foreground
-      command: |
-        pyenv install 3.7.12
-        pyenv global 3.7.12
-        sudo unlink /usr/bin/python3
-        sudo ln -s $(which python3) /usr/bin/python3
-        sudo ln -s /usr/share/pyshared/lsb_release.py /opt/pyenv/versions/3.7.12/lib/python3.7/site-packages/lsb_release.py
-
```

Comment:
Ganesh and I have discussed this new approach that makes use of the version of Python3 that is provided by the system rather than explicitly declaring one. This is in line with our current use of Java and Bazel.

We're also explicitly going to be using 
```
#! /usr/bin/env python3
```
and the command `python3` in the future, in the same way as we use `java` and `bazel`. That is, we rely upon the image that is provided to specify the appropriate version of these binaries rather than doing so ourselves.

Additionally, lsb_release is now present by default in Python 3.10